website/playground/markdown.js

```diff
@@ -1,4 +1,4 @@
-function formatMarkdown(
+export default function formatMarkdown(
```

Comment:
As far as I remember, this file cannot have `export` syntax in it, because it is also run through Node.js in some script to update the issue templare.

Reply:
@lydell, where?

Reply:
Could I change it to named export? (I haven't found the script and use cases of this file)

```js
module.exports = {
  formatMarkdown,
  getHeader,
  prettify
};
```

Reply:
I can't find it either. I think it has been removed.

Reply:
@lydell, I leave it as it is.
so, could you merge it?